ui/app/AppLayouts/Chat/stores/RootStore.qml

```diff
@@ -675,38 +675,12 @@ QtObject {
                                                                                                     d.oneToOneChatContact.displayName,
                                                                                                     d.oneToOneChatContact.alias) : ""
 
-        //Update oneToOneChatContact when the contact is updated
-        readonly property var myContactsModelConnection: Connections {
-            target: root.contactsStore.myContactsModel ?? null
-            enabled: d.activeChatType === Constants.chatType.oneToOne
+        StatusQUtils.ModelEntryChangeTracker {
```

Comment:
I'm wondering if we need the `ModelEntryChangeTracker`. In this case we can use something like this:
```
property ContactDetails oneToOneChatContact: ContactDetails {
        publicKey: d.activeChatId
        contactsStore: root.contactsStore
        profileStore: root.profileStore
}
```

Have you found other usages as well?

If this is the only usage, I'd advise against `ModelEntryChangeTracker` usage. The complete solution should be coming from the `ContactDetails` component.

Reply:
There are two usages right now: `Chat/RootStore` and `ActivityNotificationMessage`.

I fully agree it's not a final solution. Right now it's the least invasive method to unblock https://github.com/status-im/status-desktop/pull/16667 which is providing significant simplifications related to contacts models. In the next step we will remove `ModelEntryChangeTracker` which is only temporary replacement for nim's model `onItemChanged` signal, releasing dependency on that custom feature of backend's model. The target is to use `ContactDetails` or join models at early stage to provide all necessary details via models.

Regarding the `ContactDetails` in this particular context I see also some other problems we should consider:
- `ContactDetails` depends on `ProfileStore` so it would introduce unnecessary dependency to this `RootStore` even though its totally not needed here
- `ContactDetails` should not be used in store as it's breaking contract that stores are just a thin layer over backend's functionality

Reply:
Totally agree! Glad it's another step forward and not the final solution 😄 
`ContactDetails` dependencies should be simplified as well at some point if we can extract the user profile from the model and we don't need the profileStore anymore.

Ideally it would work only with a model. The purpose of the component is just to provide some default values on top of `ModelEntry`.